docs/snippets/common/msw-addon-initialize.ts.mdx

```diff
@@ -1,10 +1,10 @@
 ```ts
 // .storybook/preview.ts
 
-// Replace your-framework with the framework you are using (e.g., react, vue3)
+// Replace your-framework with the framework you are using (e.g., nextjs, vue3-vite)
 import { Preview } from '@storybook/your-framework';
```

Comment:
this is a more general feedback (in case other docs have similar structure) but the `Preview` type comes from the renderer (e.g. react, vue3), while the main.js `StorybookConfig` type comes from the frameworks